tests/functional/Codeception/Docker.php

```diff
@@ -339,10 +339,14 @@ public function addEceComposerRepo(): bool
                 $commands[] = $this->taskComposerRequire('composer')
                     ->dependency($extra['name'], $config['require'][$extra['name']])
                     ->noInteraction()
+                    ->option('--no-update')
                     ->getCommand();
             }
         }
 
+        $commands[] = $this->taskComposerUpdate('composer')
```

Comment:
This method (addEceComposerRepo () ) is always runs together with cloneTemplate() where taskComposerUpdate() also runs. Probably need to combine them in some way, e.g. remove taskComposerUpdate() from both methods and create one method which will call cloneTemplate(), addEceComposerRepo() and taskComposerUpdate()

Reply:
This should be safe, as Composer won't run subsequent `composer update`. In any case, I will add a task for this.

Reply:
@NadiyaS we can leave it as it is. After refactoring FT these functionalities will be removed.

Reply:
ok then